src/data/faq.json

```diff
@@ -1879,6 +1879,67 @@
+                        ]
+                    },
+                    {
+                        "title": "How do differences in risk assessment occur on multiple devices, even if the respective smartphones were not far apart?",
+                        "anchor": "risk_encounter_different_devices",
+                        "active": false,
+                        "textblock": [
+                            "Deviating risk assessments can have multiple causes, but do not affect the effectiveness of the Corona-Warn-App.",
+                            "The assessment of a possible risk of infection depends on the determined contact duration and distance to the smartphone of another CWA user who has decided to share his positive test result. The determination of the distance is also influenced by the orientation of the two smartphone antennas to each other. Therefore, the result of the risk calculation also depends on the exact spatial location and orientation of the smartphones involved. The environment (reflecting or absorbing surfaces) can also influence the range of the BLE signal, so that different values for contact duration and signal attenuation (distance) can be received depending on the orientation of the smartphones.",
+                            "In addition, many processes on the smartphones run in the background and can therefore start at different times. Examples of such processes are risk calculation and the loading of positive keys that takes place several times a day when WiFi access is available. But the search for other smartphones with active CWA is also such a background process.",
+                            "Furthermore, the CWA supports different versions of the Exposure Notification Framework (ENF) (depending on the version number of the app and the operating system used). This is developed and provided by Google and Apple. Since not every smartphone supports the latest version of the ENF, different ENF versions can lead to different risk calculations.",
+                            "Additionally, in the risk calculation, several low-risk encounters over the course of a day can lead to the display of an encounter with increased risk. However, this may not necessarily have been the case with another device, since it did not necessarily record the same contacts in the same period.",
+                            "In addition, due to different technical equipment (hardware) and system software alone, they react differently with regard to transmission power/signal strength and reception sensitivity. Therefore, even two adjacent devices do not necessarily receive the same diagnostic keys at the same time with identical parameters.",

```

Comment:
"they react" =>
"smartphone react"

There is no previous subject in the sentence which "they" might refer to, therefore it needs to be explicitly stated.

Reply:
Shouldn't it be "smartphones react" (plural)?

Reply:
@Ein-Tim 
Thanks for noticing the typo in my suggestion! I have corrected it now.

Reply:
Changed it in https://github.com/corona-warn-app/cwa-website/pull/2529/commits/e25bfbeb50abbc2054e6fed0564c3cd2e725c5c9, thanks!